proposals/exp-show-target.md

```diff
@@ -0,0 +1,214 @@
+│ 72ed9cd        │ Nov 15, 2020 │  0.5674 │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 50                 │ -                   │ -
+│ ├── exp-47dfa  │ Feb 17, 2021 │ 0.51625 │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                   │ -
+│ └── exp-44136  │ Feb 17, 2021 │  0.5674 │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 50                 │ -                   │ -
+│ f8e9d93        │ Nov 15, 2020 │ 0.54175 │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 50                 │ -                   │ -
+```
+
+This table view is from a simplified example and is still missing several
+columns, so this view can become cluttered quickly. Eliminating irrelevant
+information is a priority.
+
+There are several ways in which the included parameters might be irrelevant:
+* Parameters files may include values that aren't tracked by DVC at all (for
+  example, debug level).
+* The parameters may be used in a different pipeline branch (for example, the
+  `topic_modeling` stage is in a separate branch of the pipeline from the
+  `evaluate` stage, which produces the relevant experiment metrics).
+* The parameters may be used in a downstream pipeline branch (for example, the
+  post-processing `combine` stage parameters aren't relevant to experiment
+  evaluation).
+
+```console
+                +-------------------+
+                | data/data.xml.dvc |
+                +-------------------+
+                          *
+                          *
+                          *
+                     +---------+
+                     | prepare |
+                     +---------+
+                          *
+                          *
+                          *
+                    +-----------+
+                    | featurize |
+                  **+-----------+**
+              ****       *         ****
+          ****           *             ***
+        **              *                 ***
++-------+               *                    **
+| train |             **                      *
++-------+            *                        *
+         **        **                         *
+           **    **                           *
+             *  *                             *
+        +----------+                  +----------------+
+        | evaluate |                  | topic_modeling |
+        +----------+                  +----------------+
+                   ***             ****
+                      *         ***
+                       **     **
+                     +---------+
+                     | combine |
+                     +---------+
+```
+
+There are no apparent reasons that users want to see untracked parameters in
+`dvc exp show`. Unlike `dvc params diff` and `dvc metrics diff`, `dvc exp show`
```

Comment:
> There are no apparent reasons that users want to see untracked parameters in
`dvc exp show`.

This again brings me back to iterative/dvc/pull/5550. Probably that behavior should be the default, but seems like a separate question to adding `exp show [targets]`?

Reply:
see that PR is linked to close https://github.com/iterative/dvc/issues/5451. If that's the case then keeping the discussion about default behavior here makes sense, as it's closely related. (Pls resolve)